docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/paragraph--ncids-promo-block-external.html.twig

```diff
@@ -0,0 +1,59 @@
+{# ###### Setup Image Position & Image ####### #}
+{% set image_position = content.field_image_position_promo|render|striptags|lower|trim %}
```

Comment:
You have access to the paragraph entity, and thus all the fields. This is wayyyy overkill considering:
A) You can get to the data so `render/striptags` is extra work.
B) lower - this is a fixed list, so no need to lowercase. (If this were passed in as in a query parameter this would be 100% needed)
C) trim - again, the list is fixed, so unless you added extra spaces in the list, this is not needed.